config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/openstack-k8s-operators/osp-director-operator
+  newTag: 0.0.1
```

Comment:
in scripts/build_and_push_images.sh the default version is latest shouldn't this be in sync?

Reply:
I haven't actually used this script yet to build and push to the `quay.io/openstack-k8s-operators` (just against my own registry).  Once we do that, this will be properly populated.